llvm/test/CodeGen/X86/apx/xor.ll

```diff
@@ -253,7 +253,7 @@ define i16 @xor16mi(ptr %a) {
 ; CHECK-LABEL: xor16mi:
 ; CHECK:       # %bb.0: # %entry
 ; CHECK-NEXT:    movzwl (%rdi), %eax
-; CHECK-NEXT:    xorl $1234, %eax, %eax # imm = 0x4D2
+; CHECK-NEXT:    xorl $1234, %eax # imm = 0x4D2
```

Comment:
Missing test for operand swap?

Reply:
Just added llvm/test/CodeGen/X86/apx/compress-evex.mir